src/App.css

```diff
@@ -0,0 +1,93 @@
+* {
+	box-sizing: border-box;
+
+}
+
+.container {
+    padding-top: 0px;
+    display: flex;
+    background-color: black;
+    height: 110vh;
```

Comment:
generally we should avoid using vh units.  Percent units are usually better.  vh can have some weird effects on certain devices